modules/end-user-guide/partials/proc_setting-up-che-editor-yaml.adoc

```diff
@@ -1,16 +1,14 @@
 [id="setting-up-che-editor-yaml"]
 = Setting up che-editor.yaml
 
-To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file.
-
-This way, you can set a common default editor for your team and provide new contributors with the most suitable editor for your project. You can also use the `che-editor.yaml` file when you need to set a different IDE default for a particular project repository rather than the default IDE of your organization's {prod-short} instance.
+By using the `che-editor.yaml` file, you can set a common default IDE for your team and provide new contributors with the most suitable IDE for your project source code. You can also use the `che-editor.yaml` file when you need to set a different IDE default for a particular project repository rather than the default IDE of your organization's {prod-short} instance.
 
 .Procedure
 
 * In the remote Git repository of your project, create a `/.che/che-editor.yaml` file with lines that specify the relevant parameter, as described in the next section.
 
 .Verification
 
-. Visit the xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[URL for starting a new workspace].
-. Verify that the selected in-browser IDE loads in the browser tab of the started workspace.
+. xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[Start a new workspace with a clone of the Git repository].
+. Verify that the expected IDE loads in the browser tab of the started workspace.
```

Comment:
I think "selected" or "specified" sounds better here than "expected".

Reply:
Thank you for this suggestion, Jana. I'm going with "specified" as you suggested.